spec/controllers/api/v1/blood_pressures_controller_spec.rb

```diff
@@ -0,0 +1,143 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::BloodPressuresController, type: :controller do
+  describe 'POST sync: send data from device to server;' do
+
+    describe 'creates new blood pressures' do
+
+      it 'returns 400 when there are no blood pressures in the request' do
+        post(:sync_from_user, params: { blood_pressures: [] })
+        expect(response.status).to eq 400
+      end
+
+      it 'creates new blood pressures without associated patient' do
```

Comment:
Hmm... is this something we want? Creating BPs without a patient?

Reply:
This is intentional. 
BPs will have weak references to patients so can independently sync patients and blood pressures.